src/packet/ipv4.rs.in

```diff
@@ -171,7 +177,7 @@ mod checksum_tests {
 
     #[test]
     fn checksum_zeros() {
-        let mut data = vec![0u8; 20];
```

Comment:
Not sure it was ever required. I might have favored the more explicit/verbose way earlier, but now I didn't see any meaning with it since the type will be inferred. I just removed it and it worked :)